gallery_dl/extractor/instagram.py

```diff
@@ -133,11 +134,14 @@ def _extract_postpage(self, url):
             'fullname': media['owner']['full_name'],
             'post_id': media['id'],
             'post_shortcode': media['shortcode'],
+            'post_pageurl': url,
             'description': text.parse_unicode_escapes('\n'.join(
                 edge['node']['text']
                 for edge in media['edge_media_to_caption']['edges']
             )),
         }
+        common['tags'] = (
+            re.compile(r'#[^#\n ]+').findall(common['description']))
```

Comment:
Hashtags can only contain letters, numbers, and underscores from what I've read, so `r'#\w+'` would be better.

You should also pre-compile the regex in `__init__()`, so you don't have to do it for every post:
```
self._find_tags = re.compile(r'#\w+').findall
...
common['tags'] = self._find_tags(common['description'])